src/app/modules/home/container-left/container-left.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-container-left',
+  templateUrl: './container-left.component.html',
+  styleUrls: ['./container-left.component.scss']
+})
+export class ContainerLeftComponent {
+
+  constructor(
+      private router: Router
+  ) { }
+
+  public viewAllProjects() {
```

Comment:
How could the current url be `/project/overview` here? And if it was, why would you reload the page?